websites/G/Goodreads/presence.ts

```diff
@@ -0,0 +1,35 @@
+const presence = new Presence({
+	clientId: "765622778317373440"
+}),
+elapsed = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "logo-1"
+	},
+	path = window.location.href.replace(window.location.origin, ''),
+	urlParamshere = new URLSearchParams(window.location.search);
+
+	if(path === '/'){
+		presenceData.details = "Browsing Home";

```

Comment:
```suggestion
		presenceData.details = 'Browsing Home';
```

Single quotes to be more consistent.

Reply:
@SeMiD does it automatically

Reply:
We prefer "", if someone has '' @SeMiD will change it.